src/components/Editor/SearchBar.js

```diff
@@ -624,7 +624,7 @@ class SearchBar extends Component {
         {
           className: classnames(className, {
             active: !symbolSearchOn && modifiers && modifiers.get(modVal),
-            disabled: symbolSearchOn
+            hide: symbolSearchOn
```

Comment:
That seems a better way of doing it. I've modified it to not render the component. Please take a look and let me know :)